swap/src/bitcoin/wallet.rs

```diff
@@ -143,15 +143,16 @@ impl Wallet {
 
                         let new_status = match client.lock().await.status_of_script(&tx) {
                             Ok(new_status) => new_status,
-                            Err(e) => {
-                                tracing::warn!(%txid, "Failed to get status of script: {:#}", e);
+                            Err(error) => {
+                                tracing::warn!(%txid, %error, "Failed to get status of script.");
```

Comment:
We usually don't have a dot at the end of log messages.

Reply:
Scrolling through my local journal, I also don't see any dots.

Reply:
I was born and raised to use grammatically correct sentences including punctuation in log messages. 😬

Eitherway is fine with me.

Reply:
I noticed that there are some occurrences of `. ` (space after the dot) as well. Would be nice if we can clean this up. I am in favor of no dots, but that can be hard if we have multiple sentences in on message. Might be better to avoid that anyway though.

Reply:
I don't think we have any other log messages with dots so I'd be in favor to not introducing them at this stage.